blenderproc/python/loader/SuncgLoader.py

```diff
@@ -55,9 +55,9 @@ def load_suncg(house_path: str, label_mapping: LabelIdMapping,
     for level in config["levels"]:
         # Build empty level object which acts as a parent for all rooms on the level
         level_obj = create_empty("Level#" + level["id"])
-        level_obj.set_cp("type", "Level")
+        level_obj.set_cp("suncg_type", "Level")
```

Comment:
These cps are used at in the SuncgPointInRoomSampler and in CameraValidation, they also need to be renamed there

Reply:
Fixed all.